src/test/java/calculations/CalculateEnergiesTest.java

```diff
@@ -0,0 +1,58 @@
+package calculations;
+
+import image.ImageToResize;
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalculateEnergiesTest {
+    CalculateEnergies calculateEnergies = new CalculateEnergies(new ImageToResize("/seam carving image.jpg"));
+    final static int MAX_ENERGY = 6 * 255 * 255;
+
+    @Test
+    void setEnergy() {
+        //given
+        double[][] energyValues = new double[4][4];
+        Color[][] colorArray =
+                {{new Color(48, 89, 193), new Color(43, 165, 153), new Color(12, 51, 71),
+                        new Color(163, 35, 152)},
+                        {new Color(179, 193, 106), new Color(28, 34, 61), new Color(61, 48, 117),
+                                new Color(40, 30, 1)},
+                        {new Color(117, 79, 48), new Color(83, 94, 88), new Color(183, 191, 118),
+                                new Color(86, 94, 76)},
+                        {new Color(179, 193, 106), new Color(28, 34, 61), new Color(61, 48, 117),
+                                new Color(40, 30, 1)}};
+
+        //when
+        for (int i = 0; i < 4; i++) {
+            for (int j = 0; j < 4; j++) {
+                if (i + 1 >= 4 || i - 1 < 0
+                        || j + 1 >= 4 || j - 1 < 0) {
+                    energyValues[i][j] = calculateEnergies.setEnergy(MAX_ENERGY);
+                } else {
+                    energyValues[i][j] =
+                            calculateEnergies.setEnergy(colorArray[i + 1][j], colorArray[i - 1][j],
+                                    colorArray[i][j - 1], colorArray[i][j + 1]);
+
+                }
+            }
+
+        }
```

Comment:
This is the wrong way of testing this.

Reply:
I changed the parameter in the constructor for CalculateEnergies to be a Color[][]  instead of an Image - this made the test class simpler to write.